Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -364,7 +354,6 @@ cancelBooking booking mbDriver transporter = do
   QBCR.upsert bookingCancellationReason
   whenJust mbRide $ \ride -> do
     QRide.updateStatus ride.id DRide.CANCELLED
-    DLoc.updateOnRide booking.providerId ride.driverId False
```

Comment:
@chakri642 We are removing OnRide update in Location Table. Then where are we calling rideDetails with status CANCELLED ? Somewhere in a common function, can you verify if this flow would also reach to that function and it is handled already ?